NitroxClient/GameLogic/Spawning/WorldEntities/DefaultWorldEntitySpawner.cs

```diff
@@ -29,7 +29,7 @@ public IEnumerator SpawnAsync(WorldEntity entity, Optional<GameObject> parent, E
             {
                 LargeWorldEntity.Register(gameObject); // This calls SetActive on the GameObject
             }
-            else if (gameObject.GetComponent<LargeWorldEntity>() && !gameObject.transform.parent && cellRoot.IsAwake())
+            else if (gameObject.GetComponent<LargeWorldEntity>() && !gameObject.transform.parent && cellRoot.liveRoot/* && cellRoot.IsAwake()*/)
```

Comment:
Do we want this commented out IsAwake method?

Reply:
I thought I removed that...